### PR TITLE
Server info fix

### PR DESCRIPTION
**Pull request by adsnaider**
The first commit actually fixes the Server info.
The second commit only deals with formatting the Server.java file.

### CODE REVIEW

File: src/codeu/chat/client/commandline/Chat.java [hunk 1]
```diff
@@ -117,21 +117,25 @@ public void invoke(List<String> args) {
...
         if (info == null) {	
           System.out.format("ERROR: Failed to retrieve version info", args);
         } else {
           // Print the server info to the user in a pretty way
+           // Print the server info to the user in a pretty way
+          System.out.println("Server Information:");
+          System.out.format("  Start Time : %s\n", info.startTime.toString());
+          System.out.format("  Time now   : %s\n", Time.now());
+          System.out.format("  Duration   : %s sec\n", (int) (Time.duration(info.startTime,
+                Time.now()).inMs() * Math.pow(10, -3)));
```
Comment (alanhzhao) [inline src/codeu/chat/client/commandline/Chat.java]: Just divide by 1000. Cast ms to double if you want decimal.

File: src/codeu/chat/server/Server.java [hunk 2]
```diff
@@ -115,79 +112,86 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
...
     this.timeline.scheduleNow(new Runnable() {
       @Override
       public void run() {
         try {
 
           LOG.info("Reading update from relay...");
 
-          for (final Relay.Bundle bundle : relay.read(id, secret, lastSeen, 32)) {
+          for (final Relay.Bundle bundle : relay.read(id, secret, lastSeen,
+              32)) {
```
Comment (alanhzhao) [inline src/codeu/chat/server/Server.java]: fix break: keep parameters together

File: src/codeu/chat/server/Server.java [hunk 3]
```diff
@@ -247,47 +251,45 @@ private void onBundle(Relay.Bundle bundle) {
     User user = model.userById().first(relayUser.id());
 
     if (user == null) {
-      user = controller.newUser(relayUser.id(), relayUser.text(), relayUser.time());
+      user = controller.newUser(relayUser.id(), relayUser.text(),
```
Comment (alanhzhao) [inline src/codeu/chat/server/Server.java]: fix break: keep parameters together

File: src/codeu/chat/server/Server.java [hunk 4]
```diff
@@ -247,47 +251,45 @@ private void onBundle(Relay.Bundle bundle) {
...
-    ConversationHeader conversation = model.conversationById().first(relayConversation.id());
+    ConversationHeader conversation = model.conversationById()
+        .first(relayConversation.id());
 
     if (conversation == null) {
 
-      // As the relay does not tell us who made the conversation - the first person who
-      // has a message in the conversation will get ownership over this server's copy
+      // As the relay does not tell us who made the conversation - the first
+      // person who
```
Comment (alanhzhao) [inline src/codeu/chat/server/Server.java]: fix break

File: src/codeu/chat/server/Server.java [hunk 5]
```diff
@@ -247,47 +251,45 @@ private void onBundle(Relay.Bundle bundle) {
...
+      // has a message in the conversation will get ownership over this server's
+      // copy
       // of the conversation.
       conversation = controller.newConversation(relayConversation.id(),
-                                                relayConversation.text(),
-                                                user.id,
-                                                relayConversation.time());
+          relayConversation.text(), user.id, relayConversation.time());
```
Comment (alanhzhao) [inline src/codeu/chat/server/Server.java]: Also, if you must break a list of parameters into multiple lines, put one on each line.

File: src/codeu/chat/server/Server.java [hunk 6]
```diff
@@ -247,47 +251,45 @@ private void onBundle(Relay.Bundle bundle) {
...
 
     Message message = model.messageById().first(relayMessage.id());
 
     if (message == null) {
-      message = controller.newMessage(relayMessage.id(),
-                                      user.id,
-                                      conversation.id,
-                                      relayMessage.text(),
-                                      relayMessage.time());
+      message = controller.newMessage(relayMessage.id(), user.id,
```
Comment (alanhzhao) [inline src/codeu/chat/server/Server.java]: fix break: parameters

File: src/codeu/chat/server/Server.java [hunk 7]
```diff
@@ -247,47 +251,45 @@ private void onBundle(Relay.Bundle bundle) {
...
+          conversation.id, relayMessage.text(), relayMessage.time());
     }
   }
 
   private Runnable createSendToRelayEvent(final Uuid userId,
-                                          final Uuid conversationId,
-                                          final Uuid messageId) {
+      final Uuid conversationId, final Uuid messageId) {
```
Comment (alanhzhao) [inline src/codeu/chat/server/Server.java]: fix break: parameters

File: src/codeu/chat/server/Server.java [hunk 8]
```diff
@@ -247,47 +251,45 @@ private void onBundle(Relay.Bundle bundle) {
...
-        final ConversationHeader conversation = view.findConversation(conversationId);
+        final ConversationHeader conversation = view
+            .findConversation(conversationId);
         final Message message = view.findMessage(messageId);
-        relay.write(id,
-                    secret,
-                    relay.pack(user.id, user.name, user.creation),
-                    relay.pack(conversation.id, conversation.title, conversation.creation),
-                    relay.pack(message.id, message.content, message.creation));
+        relay.write(id, secret, relay.pack(user.id, user.name, user.creation),
```
Comment (alanhzhao) [inline src/codeu/chat/server/Server.java]: fix break: parameters

File: src/codeu/chat/server/Server.java [hunk 9]
```diff
@@ -66,46 +62,47 @@
...
+      public void onMessage(InputStream in, OutputStream out)
+          throws IOException {
 
         final Uuid author = Uuid.SERIALIZER.read(in);
         final Uuid conversation = Uuid.SERIALIZER.read(in);
         final String content = Serializers.STRING.read(in);
 
-        final Message message = controller.newMessage(author, conversation, content);
+        final Message message = controller.newMessage(author, conversation,
+            content);
```
Comment (alanhzhao) [inline src/codeu/chat/server/Server.java]: Don't break here. If you must break, keep parameters `author, conversation,  content ` together. Try setting your max line length to 100 characters--that's the standard at Google.

**Review by alanhzhao — CHANGES_REQUESTED** [review]

File: src/codeu/chat/server/Server.java (hunk 9 above)
Comment (adsnaider) [reply src/codeu/chat/server/Server.java]: Oh. Interesting. I always had it setup at 80 for C++ (following the google style guide). I didn't realize that they were different for java.

File: src/codeu/chat/server/Server.java (hunk 4 above)
Comment (adsnaider) [reply src/codeu/chat/server/Server.java]: Even after changing the default line wrap to 100, the formatter keeps trying to split it into three different lines...

File: src/codeu/chat/client/commandline/Chat.java (hunk 1 above)
Comment (adsnaider) [reply src/codeu/chat/client/commandline/Chat.java]: Done

File: src/codeu/chat/client/commandline/Chat.java [hunk 10]
```diff
@@ -117,21 +117,25 @@ public void invoke(List<String> args) {
...
         final ServerInfo info = context.getInfo();
         if (info == null) {	
           System.out.format("ERROR: Failed to retrieve version info", args);
         } else {
           // Print the server info to the user in a pretty way
+           // Print the server info to the user in a pretty way
+          System.out.println("Server Information:");
+          System.out.format("  Start Time : %s\n", info.startTime.toString());
+          System.out.format("  Time now   : %s\n", Time.now());
+          System.out.format("  Duration   : %s sec\n", (int) (Time.duration(info.startTime,
```
Comment (alanhzhao) [inline src/codeu/chat/client/commandline/Chat.java]: Don't cast to int. Approved for merge after fixing this.

**Review by alanhzhao — CHANGES_REQUESTED** [review]

File: src/codeu/chat/client/commandline/Chat.java (hunk 10 above)
Comment (adsnaider) [reply src/codeu/chat/client/commandline/Chat.java]: Done

**Review by alanhzhao — APPROVED** [review]

**Pull request merged**